### PR TITLE
SRVCOM-1664: ensure that docs have right context attribute and metadata, general cleanup

### PR DESCRIPTION
**Pull request by abrennan89**
Applies for OCP 4.6+

Jira: https://issues.redhat.com/browse/SRVCOM-1664

Preview links:
- https://deploy-preview-41536--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-applications#interacting-serverless-apps-http2-gRPC_serverless-applications
- https://deploy-preview-41536--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-applications#serverless-services-network-policies_serverless-applications
- https://deploy-preview-41536--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-traffic-management.html

### CODE REVIEW

**Comment by nak3** [comment]
LGTM :+1:

File: modules/odc-splitting-traffic-between-revisions-using-developer-perspective.adoc [hunk 1]
```diff
@@ -6,6 +10,11 @@ After you create a serverless application, the application is displayed in the *
 
 Any new change in the code or the service configuration creates a new revision, which is a snapshot of the code at a given time. For a service, you can manage the traffic between the revisions of the service by splitting and routing it to the different revisions as required.
 
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Serving are installed on your cluster.
+* You have logged into the {product-title} web console.
```
Comment (bobfuru) [inline modules/odc-splitting-traffic-between-revisions-using-developer-perspective.adoc:16]: Sayeth IBM SG:
```suggestion
* You have logged in to the {product-title} web console.
```

File: modules/serverless-blue-green-deploy.adoc [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
+
 :_content-type: PROCEDURE
 [id="serverless-blue-green-deploy_{context}"]
 = Routing and managing traffic by using a blue-green deployment strategy
 
 You can safely reroute traffic from a production version of an app to a new version, by using a link:https://en.wikipedia.org/wiki/Blue-green_deployment[blue-green deployment strategy].
 
+.Prerequisites
+
+* THe {ServerlessOperatorName} and Knative Serving are installed on the cluster.
```
Comment (bobfuru) [inline modules/serverless-blue-green-deploy.adoc:13]:
```suggestion
* The {ServerlessOperatorName} and Knative Serving are installed on the cluster.
```

**Review by bobfuru — COMMENTED** [review]
Couple of minor nits and otherwise LGTM!

**Pull request merged**